### DISCUSSION TITLE
Degraded performance of GitHub Actions

### DISCUSSION THREAD

**Discussion by makisukurisu** [community/community#116793] [Actions] [answered]
### Select Topic Area

Bug

### Body

Since a few minutes (13:48+ UTC, from my experience) some jobs are failing. I've noticed this with runner 16, 1 and 13 (I believe).
After a restart (and sometimes without it) Actions do work as usual.

Same happens even with self-hosted jobs:
![image](https://github.com/community/community/assets/49622129/1db5d125-9a6a-4288-889e-26d92dbb8a27)

**Comment by Storm-BE** [comment]
We're having similar issues. 
Github hosted runners are failing. Our self hosted runners work fine.

**Comment by giovannivicentin** [comment, marked as answer]
It looks like you're encountering an internal error with GitHub Actions, affecting both hosted and self-hosted jobs. This could be due to a temporary service disruption or an issue within GitHub's infrastructure. Here's what you can do to address the problem:

1. **GitHub Status**: First, check the [GitHub Status](https://www.githubstatus.com/) page to see if there is an ongoing issue with GitHub Actions or related services. If there is a reported incident, it’s likely affecting other users as well, and GitHub will be working on a fix.

2. **Restart Workflows**: You've noted that restarting sometimes helps. Continue to do this for critical workflows as a temporary measure.

3. **Check Runner Logs**: For self-hosted runners, check the runner logs for any error messages that can provide more context on why the jobs are failing.

4. **Review Workflow Configuration**: Validate your `docker.yml` configuration file to ensure there are no syntax errors or misconfigurations that could lead to sporadic failures.

5. **Contact GitHub Support**: Since this issue is occurring across multiple runners and even on self-hosted ones, reaching out to GitHub Support will be necessary. Provide them with details of the issue, including the time it started, the affected runner numbers, and screenshots or logs of the errors.

6. **Minimal Workflow Test**: Create a minimal workflow that only includes essential steps to see if it consistently passes. This could help identify if the problem is with specific actions or the overall GitHub Actions environment.

**Reply by makisukurisu** [reply]
This is the right answer, but, when I had the problem, status was green, with no reports on the site (or discussions).

**Comment by makisukurisu** [comment]
At this exact moment I have no issues, so if you don't mind I'll close this discussion thread.

Thanks for your input.
